### PR TITLE
add Yaseen549

### LINKED ISSUE #449: Yaseen549
YaseenGitHubProfile

### PR DESCRIPTION
**Pull request by Yaseen549**
resolve #449

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -767,4 +767,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
   }
+  {
+    "name": "Mohamed Yaseen Sheriff S",
+    "githubUsername": "Yaseen549",
+    "imageUrl": "https://user-images.githubusercontent.com/43935210/115549892-d377cc00-a2c6-11eb-9b4b-7983603b5c6c.jpg",
+    "issueNumber": 449
+}
```
Comment (schmelto) [inline data.json:775]:
```suggestion
  }
```
Line this up with the other curly braces

File: data.json [hunk 2]
```diff
@@ -767,4 +767,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
     "issueNumber": 445
   }
```
Comment (schmelto) [inline data.json:769]:
```suggestion
  },
```
Here is a `,` missing

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 2 above)
Comment (Yaseen549) [reply data.json]: Got it!

**Review by Yaseen549 — COMMENTED** [review]
Solved

**Review by Yaseen549 — COMMENTED** [review]
Added comma "," and removed "," after my data "},"

**Review by schmelto — APPROVED** [review]

**Review by srinath1412001 — APPROVED** [review]

**Comment by schmelto** [comment]
Hey @Yaseen549 looks like there is a merge conflict. Could you resolve this? If you need any help feel free to ask 🚀

**Comment by Yaseen549** [comment]
> Hey @Yaseen549 looks like there is a merge conflict. Could you resolve this? If you need any help feel free to ask 🚀

Hi, Thank you for informing me, but am quite new to contributing so, I don't know that much, I'll take back this commit, if it makes any problem to the repo.

**Comment by schmelto** [comment]
@Yaseen549 you don't have to close the pr. Resolving a merge conflict is no magic. You just have to pull the latest changes from this repository to your fork and then it will show you the conflicting lines. Then you can just resolve them. If you don't feel comfortable with this (you can reopen this PR) and I will solve the conflicts for you 😄

**Comment by Yaseen549** [comment]
> @Yaseen549 you don't have to close the pr. Resolving a merge conflict is no magic. You just have to pull the latest changes from this repository to your fork and then it will show you the conflicting lines. Then you can just resolve them. If you don't feel comfortable with this (you can reopen this PR) and I will solve the conflicts for you 😄

Thank you so much

**Comment by schmelto** [comment]
@Yaseen549 would you like to try resolving the conflict by your own or should I resolve the conflict?

**Comment by Yaseen549** [comment]
> @Yaseen549 would you like to try resolving the conflict by your own or should I resolve the conflict?

Can you please do this for me?

**Comment by schmelto** [comment]
@Yaseen549 done 🚀 you should find your profile nearly at the end of the readme now 👍🏼

**Comment by Yaseen549** [comment]
Thank you so much @schmelto, got it!

**Comment by schmelto** [comment]
> Thank you so much @schmelto, got it!

No problem! Nice GitHub Profile btw

**Pull request merged**